mapo/models/tests/test_policy.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for policy keras model."""
+import tensorflow as tf
+from tensorflow import keras
+from mapo.tests.mock_env import MockEnv
+from mapo.models.policy import build_deterministic_policy
+from mapo.models.layers import ActionSquashingLayer
+
+
+def test_shape():
```

Comment:
There's no need to add docstrings to test functions... It is better to incorporate a brief summary of what you're testing directly in the name of the function. For instance, instead of `test_shape()` you could simply name `test_policy_output_has_consisten_shape`..

One side benefit of doing this is that you quickly realize if tests are getting convoluted and/or checking many things at once...

Reply:
Yeah, I guess I'll just ask `pylint` to ignore the lack of docstrings in test functions in the future. Thanks for the tip.